specification/EnterpriseSupport/resource-manager/Microsoft.EnterpriseSupport/preview/2023-05-01-preview/EnterpriseSupport.json

```diff
@@ -0,0 +1,755 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/CustomParams.subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/CustomParams.resourceGroupName"
+          },
+          {
+            "name": "items",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ValidateEnterpriseSupportRequestItem"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ARM operation completed successfully.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ValidateEnterpriseSupportResponseItem"
+              },
+              "x-ms-identifiers": [],
+              "x-typespec-name": "ValidateEnterpriseSupportResponseItem[]"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ValidateEnterpriseSupport": {
+            "$ref": "./examples/ProviderLevelOperations_ValidateEnterpriseSupport.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "EnterpriseSupports": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/EnterpriseSupportsResponseProperties",
+          "description": "The resource-specific properties for this resource.",
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        }
+      },
+      "description": "A ESRPProviderHub resource",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "EnterpriseSupportsListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EnterpriseSupports"
+          },
+          "x-typespec-name": "EnterpriseSupports[]",
+          "description": "The EnterpriseSupports items on this page"
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items",
+          "x-typespec-name": "TypeSpec.Rest.ResourceLocation"
+        }
+      },
+      "description": "The response of a EnterpriseSupports list operation.",
+      "required": [
+        "value"
+      ]
+    },
+    "EnterpriseSupportsResponseProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "Provisioning status"
+        },
+        "billingAccountId": {
+          "type": "string",
+          "description": "Billing Account ID"
+        },
+        "billingGroupId": {
+          "type": "string",
+          "description": "Billing Group ID"
+        },
+        "projectId": {
```

Comment:
We should not need billingAccountId/bgId/projectId... All of that we should be able to infer from the subscription scope. If you call Create Asset directly, you can just pass in the subscriptionId directly, and Assets will resolve all of this automatically.

Reply:
this lookup of BA/BG can be achieved by calling the bigcat hydrate call using the upn. the info will be available under x-ms-bst

Reply:
we will treat this as a change and update the arm contract in August/September timeframe.